docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md

```diff
@@ -0,0 +1,300 @@
+|Interaction          | Console Interaction<br />Power on/off |
+|Provisioning         | Allow disk access<br />Allow file access<br />Allow read-only disk access<br />Allow VM download<br />Allow VM upload<br />Clone template<br />Clone VM<br />Create template from VM<br />Customize guest<br />Deploy template<br />Mark as template<br />Modify customization specifications<br />Promote disks<br />Read customization specifications |
+|Service Configuration| Allow notifications<br />Allow polling of global event notifications<br />Manage service configurations<br />Modify service configurations<br />Query service configurations<br />Read service configurations |
+| Snapshot Management  | Create snapshot<br />Remove snapshot<br />Rename snapshot<br />Revert to snapshot |
+|vSphere Replication  | Configure replication<br />Manage replication<br />Monitor replication |
+| vSAN  | Cluster<br />ShallowRekey |
+
+</TabItem>
+
+
+</Tabs>
+
+
+
+## Zone Tagging
+You can use tags to create node zones and regions for your Kubernetes clusters. The node zones and regions can be used to dynamically place Kubernetes workloads and achiveve higher availability. Kubernetes nodes inherit the zone and region tags as [Labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/). Kubernetes workloads can use the node labels to ensure that the workloads are deployed to the correct zone and region.
+
+The following is an example of node labels that are discovered and inherited from vSphere tags. The tag values are applied to Kubernetes nodes in vSphere.
+
+```yaml hideClipboard
+  topology.kubernetes.io/region=usdc
+  topology.kubernetes.io/zone=zone3
+  failure-domain.beta.kubernetes.io/region=usdc
+  failure-domain.beta.kubernetes.io/zone=zone3 
+```
+
+
+:::info
+
+To learn more about node zones and regions, refer to the [Node Zones/Regions Topology](https://cloud-provider-vsphere.sigs.k8s.io/cloud_provider_interface.html) section of the Cloud Provider Interface documentation.
+
+:::
+
+
+Zone tagging is required to install Palette and is helpful for Kubernetes workloads deployed in vSphere clusters through Palette if they have persistent storage needs. Use vSphere tags on data centers and compute clusters to create distinct zones in your environment. You can use vSphere [Tag Categories and Tags](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-vcenter-esxi-management/GUID-16422FF7-235B-4A44-92E2-532F6AED0923.html) to create zones in your vSphere environment and assign them to vSphere objects.
+
+
+The zone tags you assign to your vSphere objects, such as a datacenter and clusters are applied to the Kubernetes nodes you deploy through Palette into your vSphere environment. Kubernetes clusters deployed to other infrastructure providers, such as public cloud may have other native mechanisms for auto discovery of zones.
```

Comment:
We don't have "data center" on our word list and should add it. There's a mix of single-word and two-word occurrences in this section. I think we decided on two words, as it appears that way in our nav pane. Let's confirm with each other in our team meeting, and I'll add it to the word list.